components/FeedbackModal.js

```diff
@@ -132,9 +132,11 @@ export default function FeedbackModal() {
           break
         case submissionSubject:
           feedbackData.message = `Venue ID: ${formData.venueId}\nSubmission ID: ${formData.submissionId}\n\n${formData.message}`
+          feedbackData.subject = `${formData.subject} - ${formData.venueId} - ${formData.submissionId}`
```

Comment:
Personally I don't think having the text "A conference I submitted to" in the subject is necessary. I would change this to:

```suggestion
          feedbackData.subject = `${formData.venueId} - ${formData.submissionId}`
```

Reply:
thanks I have misread your change
please take a look at 
[97bae8f](https://github.com/openreview/openreview-web/pull/1871/commits/97bae8f784abae322a4c267f0c9a65945d739c45)

i think having submission id itself is already unique